.github/workflows/ci.yml

```diff
@@ -0,0 +1,42 @@
+on: [push, pull_request]
+name: ci
+env:
+  GO111MODULE: on
+
+jobs:
+  test:
```

Comment:
perhaps name validate or validate-change (it does more than test)

Reply:
Sure I can change it to validate. I think the free tier should be more the enough, since it includes 2,000 minutes for free.